### ISSUE TITLE
Bundle mkxp with a CJK font like WenQuanYi Micro Hei ?

### ISSUE THREAD

**Issue by maz-1** [Ancurio/mkxp#135]
There are quite a lot rpgmaker games by asian authors, so bundle mkxp with a CJK font should be a good idea. Consider WenQuanYi Micro Hei, it is small (about 4 MB) and contains enough characters for most rpgmaker games.

**Comment by Ancurio** [comment]
I have considered doing this in the past, but distributing executables bloated up by 4 megs of static data 95% of mkxp users will never require doesn't seem very sound.

It's easy enough to change the bundled font yourself to whatever you want, or to bundle external fonts with your game (see Readme) if you don't want to compile yourself.

**Comment by maz-1** [comment]
Yeah, but I still think it would be good to be able to set a fallback font in the configuration file, so I don't need to modify the game script myself.

**Comment by Ancurio** [comment]
Having a conf entry to change the global fallback font would be a functionality that I'd be okay with.

> I guess this is what "FontSub" is trying to do?

No. Please read the documentation in mkxp.conf.sample.
